.github/workflows/unit.yml

```diff
@@ -1,22 +0,0 @@
-# Run unit tests that don't require secrets on any branch/fork pull request
-on:
-  pull_request:
-name: Unit tests
-
-jobs:
-  unit:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        target:
-          - docs-check
```

Comment:
so where is it checked now? because the contents of tools.go are different than docs rule in Makefile

Reply:
in the docs-and-format.yml workflow

Reply:
Missed that